.github/workflows/sample-distribution.yml

```diff
@@ -34,24 +32,15 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pods-
       - name: iOS Pods setup
-        uses: nick-invision/retry@v2
-        with:
-          timeout_minutes: 30
-          max_attempts: 3
-          command: |
-            cd examples/SampleApp
-            cd ios
-            bundle exec pod install
```

Comment:
Regarding this - I have not faced this issue during my testing, but if for some reason it still reproduces, I'd prefer to create a tiny custom repeater instead of this third-party action.